var/spack/packages/vim/package.py

```diff
@@ -44,6 +50,11 @@ def install(self, spec, prefix):
               feature_set,
               fs))
           feature_set = fs
+      if '+gui' in spec:
+        if feature_set is not None:
+          if feature_set is not 'huge':
+            tty.error("+gui variant requires 'huge' feature set, {} was specified".format(feature_set))
```

Comment:
@alfredo-gimenez: You can't use {} in a format expression in 2.6 -- this should break one of the tests so be sure to run `spack test` before committing and it'll catch this.  I'll merge and fix this time.